server/jest-setup.js

```diff
@@ -3,6 +3,9 @@ const replaySetup = require('../test/helpers/replay-setup')
 
 replaySetup('success')
 
+// The pipeline complains at the default of 5s - so double it!
```

Comment:
Might be useful to comment what is meant by default. or something like `//This increases the interval between the before and after hooks and tests from 5s to 10s.

Reviewers see:
https://jestjs.io/docs/en/jest-object#jestsettimeouttimeout

Reply:
Ooops - I did plan on putting that link in the comment - let me do that... 

What do you mean by:
> Might be useful to comment what is meant by default

Reply:
As in what the time relates to. I suppose the docs link would be sufficient explanation

Reply:
Added the link ... the other I think is a matter of style so will leave the rest of the comment intact